data-raw/create_adams_data.R

```diff
@@ -0,0 +1,167 @@
+# ensure every packages are installed
+
+update_pkg <- FALSE
+ignore_templates <- list(
+  "admiralonco" = c("ad_adrs_basic.R")
+)
+
+save_rda <- function(data, file_path, new_name) {
+  if (missing(new_name)) {
+    save(data, file = file_path, compress = "bzip2")
+  } else {
+    assign(new_name, data)
+    save(list = new_name, file = file_path, compress = "bzip2")
+  }
+}
+
+load_rda <- function(fileName) {
+  load(fileName)
+  get(ls()[ls() != "fileName"])
+}
+
+write_doc <- function(data, dataset_name) {
+  # create documentation for the current dataset
+  # TODO: use metatools/metacore for doc  ?
+  doc_string <- paste(
+    sprintf("#' Dataset %s", dataset_name),
+    "#'",
+    sprintf("#' %s dataset", dataset_name),
+    "#'",
+    sprintf("#' @name %s", dataset_name),
+    "#' @docType data",
+    sprintf("#' @format A data frame with %s columns:", ncol(data)),
+    "#'   \\describe{",
+    paste(sapply(names(data), function(col_name) {
+      paste(sprintf("#'     \\item{ %s }{%s}", col_name, col_name))
```

Comment:
Yes good point it's something we discussed with @manciniedoardo (I think I can embed this with Stefen suggestion (https://github.com/pharmaverse/pharmaverseadam/pull/53#discussion_r1433947662) )

Reply:
@kaz462 @dgrassellyb the labels have all been compiled in [the shared gsheet](url). is it possible to now download this sheet as an excel file and upload it to the package in this PR so that we can both populate the datasets with labels and make sure the labels appear on the website with this PR? Or are there added dificulties?

Reply:
Hi @manciniedoardo I updated the script (but didn't push the updates yet, need to test) - but I can not access the gsheet (your link is not working) - could you please send me the gsheet to be able to get latest labels ?

Reply:
Hi @dgrassellyb apologies, shared with you now.